pom.xml

```diff
@@ -373,6 +398,21 @@
       <artifactId>jclouds-allblobstore</artifactId>
       <version>${jclouds.version}</version>
     </dependency>
+    <dependency>
```

Comment:
had to add these otherwise I can't reference them in the shade plugin

Reply:
This section confuses me -- what is going on here?

Reply:
the jclouds dependences were pulled in transitively, the maven shade plugin doesn't pick them up, so I had to add them explicitly

Reply:
Should this include the the entire tree of dependencies, e.g., b2, google-cloud-storage, openstack-swift, etc.?

Reply:
:) that is a good question and it depends. If those jars use guava they need to be added to the list of shading deps and added to the pom. If they don't and I suspect we're ok for it, they don't need to be added.

Just to clarify I've only added the jcloud jars so I can reference them in the mvn shade plugin:
`<include>org.apache.jclouds:*</include>`

Reply:
Just doing jclouds resolves our downstream issue. I guess we could start with just this and expand it later as/when necessary?